### DISCUSSION TITLE
How can I see in an IDE, linting errors that Deno doesn't catch?

### DISCUSSION THREAD

**Discussion by dandv** [denoland/deno#27493] [Q&A]
Consider this code:
```ts
const x = Math.random();

if (x < 0.5)
  if (x < 0.1)
    console.log('x is really small', x);
else
  console.log('x > 0.5', x);
```

Neither `deno lint`, nor `biome`, nor `oxlint` had any complaints about it by default, even though the error is quite obvious.

I don't want to opt into automatic formatting so, I *could* run `deno fmt --check` and try to ignore everything except that issue with the `else`, but this isn't realistic and doesn't integrate with the LSP either.

This is a situation in which a formatting rule ([eslint's `curly`](https://eslint.org/docs/latest/rules/curly)) would catch a logical error.

**Comment by marvinhagemeister** [comment]
I think you're asking for eslint's `curly` rule to be added to Deno's linter. Made a PR here: https://github.com/denoland/deno_lint/pull/1383
